scripts/linters/pylint_extensions.py

```diff
@@ -81,8 +81,98 @@ class ExplicitKeywordArgsChecker(checkers.BaseChecker):
             'non-explicit-keyword-args',
             'All keyword arguments should be explicitly named in function call.'
         ),
+        'C0027': (
+            'Keyword argument %s used for a non keyword argument in %s '
+            'call of %s.',
+            'arg-name-for-non-keyword-arg',
+            'Position arguments should not be used as keyword arguments '
+            'in function call.'
+        ),
     }
 
+    def _check_non_explicit_keyword_args(
+            self, node, name, callable_name, keyword_args,
+            num_positional_args_unused, num_mandatory_parameters):
+        """Custom pylint check to ensure that position arguments should not
+        be used as keyword arguments.
+
+        Args:
+            node: astroid.node.Function. The current function call node.
+            name: str. Name of the keyword argument.
+            callable_name: str. Name of method type.
+            keyword_args: list(str). Name of all keyword arguments in function
+                call.
+            num_positional_args_unused: int. Number of unused positional
+                arguments.
+            num_mandatory_parameters: int. Number of mandatory parameters.
+
+        Returns:
+            int. Number of unused positional arguments.
+        """
+        display_name = repr(name)
+
+        if name not in keyword_args and (
+                num_positional_args_unused > (
+                    num_mandatory_parameters)) and (
+                        callable_name != 'constructor'):
+            # This try/except block tries to get the function
+            # name. Since each node may differ, multiple
+            # blocks have been used.
+            try:
+                func_name = node.func.attrname
+            except AttributeError:
+                func_name = node.func.name
+
+            self.add_message(
+                'non-explicit-keyword-args', node=node,
+                args=(
+                    display_name,
+                    callable_name,
+                    func_name))
+            num_positional_args_unused -= 1
+        return num_positional_args_unused
+
+    def _check_argname_for_nonkeyword_arg(
+            self, node, called, callable_name, keyword_args,
+            keyword_args_in_funcdef):
+        """Custom pylint check to ensure that position arguments should not
+        be used as keyword arguments.
+
+        Args:
+            node: astroid.node.Function. The current function call node.
+            called: astroid.Call. The function call object.
+            keyword_args: list(str). Name of all keyword arguments in function
+                call.
+            callable_name: str. Name of method type.
+            keyword_args_in_funcdef: list(str). Name of all keyword arguments in
+                function definition.
+        """
+        for arg in keyword_args:
+            # TODO(Hudda): Fix the check to cover below case as well. Relevant
```

Comment:
The correct format for TODOs is `TODO(#10038)`. Please fix this in your next PR.

Reply:
Also please reply here with a link to the change when it's done.

Reply:
The todo comment is changed now. Here is the link to the change https://github.com/oppia/oppia/blob/f4b140265d8d0c65e9887b393558325a6c3bd624/scripts/linters/pylint_extensions.py#L148

Reply:
Thanks, @Hudda! Appreciate the follow-up.